upstream-community-operators/sysdig/sysdig-operator.v1.4.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,100 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: sysdig-operator.v1.4.0
+  namespace: placeholder
+  annotations:
```

Comment:
Please add `metadata.annotations.alm-examples` to this CSV. The value should be the string representation of a JSON list of an example CR for the operator's CRD.